_posts/ja/newsletters/2024-01-31-newsletter.md

```diff
@@ -0,0 +1,209 @@
+  「コンセンサスルールはブロックハッシュによってコミットされた情報のみに依存できる」理由について説明しています。
+
+- [Sigop数とそれがトランザクションの選択に与える影響は？]({{bse}}121355)
+  ユーザーCosmik Debrisは、署名チェック操作「sigops」の制限がマイナーのブロックテンプレートの構築と
+  mempoolベースの[手数料の推定][topic fee estimation]にどのような影響を与えるかを尋ねています。
+  ユーザーmononautは、ブロックテンプレートの構築における制限要因であるsigopsの頻度の低さについて説明し、
+  `-bytespersigop`オプションについて論じています。
+
+## リリースとリリース候補
+
+*人気のBitcoinインフラストラクチャプロジェクトの新しいリリースとリリース候補。
+新しいリリースにアップグレードしたり、リリース候補のテストを支援することを検討してください。*
+
+- [HWI 2.4.0][]は、複数の異なるハードウェア署名デバイスへの共通のインターフェースを提供する
+  このパッケージの次期バージョンのリリースです。この新しいリリースでは、
+  Trezor Safe 3のサポートが追加され、いくつかの小規模な改善が含まれています。
+
+## 注目すべきコードとドキュメントの変更
+
+*今週の[Bitcoin Core][bitcoin core repo]、[Core
+Lightning][core lightning repo]、[Eclair][eclair repo]、[LDK][ldk repo]、
+[LND][lnd repo]、[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet
+Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay
+Server][btcpay server repo]、[BDK][bdk repo]、[Bitcoin Improvement
+Proposals（BIP）][bips repo]、[Lightning BOLTs][bolts repo]、
+[Bitcoin Inquisition][bitcoin inquisition repo]および[BINANAs][binana repo]の注目すべき変更点。*
+
+- [Bitcoin Core #29291][]では、`OP_CHECKSEQUENCEVERIFY` opcodeを実行するトランザクションの
+  バージョン番号が負であると思われる場合に失敗するテストが追加されています。
+  このテストが、代替のコンセンサス実装で実行されていれば、
+  [先週のニュースレター][news286 bip68ver]で言及したコンセンサス障害のバグは発見できたでしょう。
+
+- [Eclair #2811][]、[#2813][eclair #2813]、[#2814][eclair #2814]は、
+  [トランポリンペイメント][topic trampoline payments]で最終的な受信者に対して
+  [ブラインドパス][topic rv routing]を使用する機能を追加しました。
+  トランポリンルーティング自体は、通常のOnion暗号化したノードIDを引き続き使用します。
+  つまり、各トランポリンノードは次のトランポリンノードのIDを学習します。
+  ただし、ブラインドパスが使用されている場合、最後のトランポリンノードは、
+  ブラインドパス内の最初の転送ノードのノードIDのみを学習します。
+  最終的な受信者のノードIDは学習されません。
+
+    以前は、強力なトランポリンプライバシーは、複数のトランポリン転送者の使用に依存していたため、
+    どの転送者も自分が最後の転送者であることを確信できませんでした。
+    このアプローチの欠点は、より長いパスが使用されるため、転送が失敗する確率が高まり、
+    成功のためにより多くの転送手数料を支払う必要があることです。
+    現在は、単一のトランポリンノードを介した支払いの転送でも、
+    そのノードが最終的な受信者を学習するのを防止できます。
+
+- [LND #8167][]は、LNDチャネルが、まだ1つ以上の支払い（[HTLC][topic htlc]）の保留があるチャネルの協調クローズを可能にします。
+  [BOLT2][]の仕様では、この場合の適切な手順は、一方が`shutdown`メッセージを送信し、
+  それ以降新しいHTLCが受け入れられなくなると記載されています。すべての保留中のHTLCがオフチェーンで解決された後、
+  両者は交渉して協調クローズトランザクションに署名します。以前は、
+  LNDが`shutdown`メッセージを受信すると、チャネルが強制的に閉じられ、
+  決済のために追加のオンチェーントランザクションと手数料が必要でした。
+
+- [LND #7733][]は、[ウォッチタワー][topic watchtowers]のサポートを更新し、
+  現在LNDによって実験的にサポートされている[Simple Taproot Channel][topic simple taproot channels]の
+  バックアップと正しいシャットダウンを強制できるようになりました。
+
+- [LND #8275][]は、[BOLTs #1092][]で定義されているように、
+  ピアが特定の普遍的にデプロイされた機能（[ニュースレター #259][news259 lncleanup]参照）をサポートすることを要求するようになりました。
+
+- [Rust Bitcoin #2366][]は、`Transaction`オブジェクトの`.txid`プロパティを廃止し、
+  代わりに`.compute_txid`という名前のプロパティを提供するようになりました。
+  `.txid`プロパティが呼ばれるたびに、トランザクションのtxidが計算され、
+  これは十分なCPUを消費するため、大きなトランザクションや多数の小さなトランザクションで関数を実行している人にとっては問題になります。
+  このプロパティの新しい名前は、下流のプログラマーがその潜在的なコストを認識するのに役立つと期待されています。
+  `.wtxid`プロパティと`.ntxid`プロパティ（それぞれ[BIP141][]と[BIP140][]に基づく）も同様に、
+  `.compute_wtxid`と`.compute_ntxid`にリネームされています。
+
+- [HWI #716][]は、[Trezor Safe 3][]ハードウェア署名デバイスのサポートを追加しました。
```

Comment:
```suggestion
- [HWI #716][]は、ハードウェア署名デバイスのサポートを追加しました。
```

Reply:
@bitschmidty thanks. I removed the link.